chromium_src/chrome/browser/metrics/chrome_metrics_services_manager_client.h

```diff
@@ -0,0 +1,26 @@
+/* Copyright (c) 2020 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * you can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#ifndef BRAVE_CHROMIUM_SRC_CHROME_BROWSER_METRICS_CHROME_METRICS_SERVICES_MANAGER_CLIENT_H_
+#define BRAVE_CHROMIUM_SRC_CHROME_BROWSER_METRICS_CHROME_METRICS_SERVICES_MANAGER_CLIENT_H_
+
+#define ChromeMetricsServicesManagerClient \
```

Comment:
We could probably just rename the method instead of renaming the entire class?
<pre>
#define IsMetricsReportingEnabled IsMetricsReportingEnabled_ChromiumImpl(); \
  bool IsMetricsReportingEnabled
</pre>

Reply:
ok, I rename the method only. PTAL.

Reply:
Thanks, it LGTM, just want to ping @jumde to make sure this fulfills the initial intent of https://github.com/brave/brave-core/pull/2095.

Reply:
lgtm, please verify that `npm run network-audit` passes CI on all platforms

Reply:
ok, let me check it on all platforms.